devsearch/preprocessing/extract.py

```diff
@@ -0,0 +1,176 @@
+from collections import defaultdict, Counter
+import difflib
+import json
+import os
+from pprint import pprint
+import time
+from typing import Any, Dict, Iterator, List, Union
+
+from dotenv import load_dotenv
```

Comment:
it's not included into commit

Reply:
let's improve it - you can require key in CLI and/or provide explanation how to use environment variable